kbase-extension/static/kbase/js/widgets/appWidgets2/fieldWidgetCompact.js

```diff
@@ -344,20 +365,24 @@ define([
                         }),
                         div({
                             id: ids.feedback,
-                            class: 'input-group-addon',
+                            class: 'input-group-addon kb-input-group-addon kb-app-field-feedback',
                             dataElement: 'feedback',
                             style: {
-                                width: '30px',
-                                padding: '0'
+                                width: '3px',
+                                height: '100%',
+                                'margin-left': '4px'
                             }
                         }, [
                             div({
                                 id: ids.feedbackIndicator,
-                                dataElement: 'indicator'
+                                dataElement: 'indicator',
+                                style: {
+                                    width: '3px'
+                                }
                             })
                         ]),
-                        div({
-                            class: 'input-group-addon',
+                        /*div({
```

Comment:
Yes- all these blocks should be removed if we are going to stick with removing the current 'info' button.  I probably can't get to it though until much later this afternoon or tomorrow morning though...